kitsune/sumo/tests/test_templates.py

```diff
@@ -1,22 +1,19 @@
 from django.template import engines as template_engines
```

Comment:
Seems like black touched this file for some reason, can we remove these changes from this commit?

Reply:
this is not from black. Just some (automatic) sorting of the import modules